include/settings.h

```diff
@@ -32,6 +32,7 @@ typedef struct
     uint8_t doSort;
     uint8_t doAni;
     uint8_t update;
+    uint8_t prompt_fade;
```

Comment:
I wonder if users would want an additional setting for the footer, or maybe the button fade is just part of the `Animations` setting? 🤔

Reply:
For me at least, I wanted buttons to stay on screen with animations disabled. I always had to press a button to see the prompts again so I added this

Reply:
I see, mmm... then let's just keep a single setting "Animations" and also use it for the help footer fade-out.

Honestly I don't use the footer as reference because I already know what I want to do 😅  , and the fade-out is a legacy from the original Artemis PS3 code.